tests/tck/features/expression/function/TypeConversion.feature

```diff
@@ -9,72 +9,81 @@ Feature: TypeConversion Expression
   Scenario: toBoolean
     When executing query:
       """
-      YIELD [toBoolean(true), toBoolean(false), toBoolean(1), toBoolean(3.14),
+      YIELD [toBoolean(true), toBoolean(false),
       toBoolean("trUe"), toBoolean("3.14"), toBoolean(null)] AS yield_toBoolean
       """
     Then the result should be, in any order:
-      | yield_toBoolean                                     |
-      | [true, false, BAD_TYPE, BAD_TYPE, true, NULL, NULL] |
+      | yield_toBoolean                 |
+      | [true, false, true, NULL, NULL] |
     When executing query:
       """
-      UNWIND [true, false, 1, 3.14, "trUe", "3.14", null] AS b
+      UNWIND [true, false, "trUe", "3.14", null] AS b
       RETURN toBoolean(b) AS unwind_toBoolean
       """
     Then the result should be, in any order:
       | unwind_toBoolean |
       | true             |
       | false            |
-      | BAD_TYPE         |
-      | BAD_TYPE         |
       | true             |
       | NULL             |
       | NULL             |
+    When executing query:
+      """
+      YIELD [toBoolean(1), toBoolean(3.14)] AS yield_toBoolean
+      """
+    Then a SemanticError should be raised at runtime: Type error `toBoolean(1)'
 
   Scenario: toFloat
     When executing query:
       """
-      YIELD [toFloat(true), toFloat(false), toFloat(1), toFloat(3.14),
+      YIELD [toFloat(1), toFloat(3.14),
       toFloat("trUe"), toFloat("3.14"), toFloat(null)] AS yield_toFloat
       """
     Then the result should be, in any order:
-      | yield_toFloat                                     |
-      | [BAD_TYPE, BAD_TYPE, 1.0, 3.14, NULL, 3.14, NULL] |
+      | yield_toFloat                 |
+      | [1.0, 3.14, NULL, 3.14, NULL] |
     When executing query:
       """
-      UNWIND [true, false, 1, 3.14, "trUe", "3.14", null] AS b
+      UNWIND [1, 3.14, "trUe", "3.14", null] AS b
       RETURN toFloat(b) AS unwind_toFloat
       """
     Then the result should be, in any order:
       | unwind_toFloat |
-      | BAD_TYPE       |
-      | BAD_TYPE       |
       | 1.0            |
       | 3.14           |
       | NULL           |
       | 3.14           |
       | NULL           |
+    When executing query:
+      """
+      YIELD [toFloat(true), toFloat(false)] AS yield_toFloat
+      """
+    Then a SemanticError should be raised at runtime: Type error `toFloat(true)'
 
   Scenario: toInteger
     When executing query:
       """
-      YIELD [toInteger(true), toInteger(false), toInteger(1), toInteger(3.14),
```

Comment:
Why change this case?

Reply:
toInteger(false), toInteger(true) will report error in opencypher,   other modified cases too

Reply:
I think you should keep the origin case, and add new one to test the origin function.

Reply:
i know, but the result of origin case is wrong, and the modified logic test has been covered